src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -254,4 +255,13 @@ public String getJavaHomeDir() {
   public void setJavaHomeDir(String javaHomeDir) {
     this.javaHomeDir = javaHomeDir;
   }
+
+  @Override
+  public Boolean getClearDatastore() {
+    return clearDatastore;
+  }
+
+  public void setClearDatastore(Boolean clearDatastore) {
```

Comment:
We haven't been documenting value class getters/setters like this in this project. Do you think we should?